hudi-utilities/src/main/java/org/apache/hudi/utilities/streamer/StreamSync.java

```diff
@@ -1037,6 +1037,11 @@ public void runMetaSync() {
 
       Map<String, HoodieException> failedMetaSyncs = new HashMap<>();
       for (String impl : syncClientToolClasses) {
+        if (impl.trim().isEmpty()) {
+          LOG.warn("Cannot run MetaSync with empty class name");
```

Comment:
Let's add a small test case so that the test fails before the fix and passes after the fix.  Do you need to fix other places?  Also seems like this change is already in #11915?

Reply:
Then this is a mistake, because I thought I did not submitted it yet.

Reply:
I will try to add a test for this in a separate PR.